DatadogSessionReplay/SRSnapshotTests/SRHost/Fixtures/BasicViewControllers.swift

```diff
@@ -16,6 +16,15 @@ internal class ShapesViewController: UIViewController {
     }
 }
 
+internal class TextsViewController: UIViewController {
+    @IBOutlet weak var textView: UITextView?
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        textView?.becomeFirstResponder()
```

Comment:
question/ If this makes the carriage to appear on snapshot image (left-hand-side), won't it be causing flakiness if the actual app screenshot is captured w/ vs w/o the carriage?